docs/source/installation/installation.rst

```diff
@@ -9,27 +9,25 @@ Installation
 Manual Setup
 ============
 
-**1. Set environment variables**
+The plugin is a Maya module. To use it, simply uncompress the zip file and add the path containing the :file:`appleseed-maya.mod` file to your `Maya.env file <https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2016/ENU/Maya/files/GUID-8EFB1AC1-ED7D-4099-9EEE-624097872C04-htm.html>`_, or to the `MAYA_MODULE_PATH environment variable <https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2015/ENU/Maya/files/Environment-Variables-File-path-variables-htm.html>`_.
```

Comment:
I updated the installation instructions in the release notes, can you grab them?